internal/services/eventhub/eventhub_resource.go

```diff
@@ -237,6 +237,7 @@ func resourceEventHubUpdate(d *pluginsdk.ResourceData, meta interface{}) error {
 			PartitionCount:         utils.Int64(int64(d.Get("partition_count").(int))),
 			MessageRetentionInDays: utils.Int64(int64(d.Get("message_retention").(int))),
 			Status:                 &eventhubStatus,
+			CaptureDescription:     expandEventHubCaptureDescription(d),
```

Comment:
Does this always need to be set for an update? i.e. if `nil` is sent, is the value removed from the resource? (Which would indicate a bug in the service?)

Reply:
I tried using the api directly, the value would be removed if I didn't include this property. I think it makes sense since we are not explicitly setting it when sending the request. 

To answer your questions, if we set the value to nil, the value will be removed as well, doesn't it sounds an expected behavior to you? feel free to let me know your concerns. :)

Reply:
👍